src/components/DailyDataDisplay/DailyDataDisplay.js

```diff
@@ -20,11 +20,11 @@ function DailyDataDisplay({region, rasterMetadata, model, emission}){
   const [climatology, setClimatology] = useState(null);
 
   function selectVariable(event) {
-      setVariable(event.value);
+      setVariable(event);
   }
   
   function selectClimatology(event) {
-      setClimatology(event.value);
+      setClimatology(event);
   }
```

Comment:
I'm guessing that these functions preserve the `select*` naming you use, which is a nice touch. But they just rename the function, so you could just do `const selectX = setX` or even just use the `select*` name as the output of the `useState`.